sprint1/sprint1.ino

```diff
@@ -1,5 +1,14 @@
 #include <Smartcar.h>
 #include <HardwareSerial.h>
+#include <vector>
```

Comment:
Why is this sketch here? Is it used? If not then remove it. If yes, then you **must** build it on CI.